README.md

```diff
@@ -304,6 +304,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 
 ## Groupware
 
+  * [Cozy Cloud](https://cozy.io) - A personal cloud where you can read your emails or manage and sync your contact, files or calendars, with an app store full of community contributions - `GPLv3`
```

Comment:
Can you add ([Source code](https://github.com/cozy))? As well as `Nodejs` for the platform/language